### ISSUE TITLE
A11y: Insufficient Link Context

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#3195]
## Summary

The link that is displayed when a user has no Language Information on their profile does not provide sufficient context to users that may be accessing the site using assistive technology.

## Screenshot

<img width="567" alt="Screen Shot 2022-06-28 at 8 56 30 AM" src="https://user-images.githubusercontent.com/4127998/176184328-b6890f79-dcb9-4fcf-81cc-05ac936d9645.png">

## Acceptance Criteria

- [ ] Update link text so that it does not need external context to understand where the link will direct a user

**Comment by patcon** [comment]
Whether to do the above is maybe a more a general solicitation of feedback from devs. it has technical implications, so not sure if it's worthwhile or not. So take the above string suggestion with a grain of salt :)

But given all the different strings we have for variations on the same theme, it might be nice to figure out a way to make the above style work for us.

**Comment by gobyrne** [comment]
@patcon why is the text I suggested above problematic?

**Comment by patcon** [comment]
No no it's not problematic in your transposing it :) but just that I'm admitting that my suggestion in Slack (which you correctly heard) is maybe tougher than it seems (technically), so I was just outing myself as the "source" of the requirement.

It's not a problem, just maybe a separate discussion we could kick down the road, if it's non-trivial :)
